macros/EmbedTest262ReportResultsTable.ejs

```diff
@@ -0,0 +1,12 @@
+<%
+// Embeds a Test262.report results table
+//
+// Parameters:
+//  $0 - A "feature" name from test262.report
+//
+//  Example calls:
+// {{EmbedTest262ReportResultsTable("globalThis")}}
+// {{EmbedTest262ReportResultsTable("BigInt")}}
+//
+%>
+<iframe frameborder="0" scrolling="no" style="width: 100%; height: 300px;" src="https://test262.report/embed/features/<%= $0 %>?engines=chakra%2Cjavascriptcore%2Cspidermonkey%2Cv8&summary=true&include-browsers=true"></iframe>
```

Comment:
Actually, we need URI escaping for this, instead of HTML escaping, which is what `<%=` does:

```suggestion
<iframe
    frameborder="0" scrolling="no" style="width: 100%; height: 300px;"
    src="https://test262.report/embed/features/<%-
        encodeURIComponent($0)
    %>?engines=chakra%2Cjavascriptcore%2Cspidermonkey%2Cv8&summary=true&include-browsers=true"
></iframe>
```

Reply:
@ExE-Boss I don't mind making this change, but I was curious to see how this is used in other macros, so I did a search and no other macros are using `encodeURIComponent`—is that a new requirement?